test/unit/node-labels.test.js

```diff
@@ -235,6 +235,39 @@ tap.test('label: "lib / src" when 5 or more JS sub-systems have been changed', (
   t.end()
 })
 
+// https://github.com/nodejs/node/pull/12366 should have been labelled "lib / src"
+// https://github.com/nodejs/github-bot/issues/137
+tap.test('label: "lib / src" when 5 or more native files have been changed', (t) => {
```

Comment:
maybe I’m not understanding the above code, but doesn’t that happen for *4* or more files? :smile: